### PR TITLE
PEP: pandas/core final round (api, matrix, sparse, cleanup of common,…

### PR DESCRIPTION
**Pull request by rockg**
… generic)

Besides groupby we're clean.

### CODE REVIEW

File: pandas/core/matrix.py [hunk 1]
```diff
@@ -1 +1,3 @@
+# flake8: noqa
+
```
Comment (jreback) [inline pandas/core/matrix.py]: hmm this is a really really old file for compat

Comment (wesm) [inline pandas/core/matrix.py]: should slate for removal, or at least start raising a deprecation warning when people import this file

**Comment by wesm** [comment]
lgtm

**Comment by wesm** [comment]
cherry-picked, thank you!

**Pull request closed without merging**